src/utilstrencodings.h

```diff
@@ -44,6 +44,10 @@ std::vector<unsigned char> ParseHex(const char* psz);
 std::vector<unsigned char> ParseHex(const std::string& str);
 signed char HexDigit(char c);
 bool IsHex(const std::string& str);
+std::vector<unsigned char> DecodeAscii85(const char* p, bool* pfInvalid = NULL);
+std::string DecodeAscii85(const std::string& str);
+std::string EncodeAscii85(const unsigned char* pch, size_t len);
+std::string EncodeAscii85(const std::string& str);
 std::vector<unsigned char> DecodeBase64(const char* p, bool* pfInvalid = NULL);
```

Comment:
nullptr
Also, if functions are not throwing exceptions, but rather returns error code (like in your case) - mark it noexcept. Compiler will generate much faster code in this case.